frontend/src/addEventPage/AddEventForm.js

```diff
@@ -50,7 +45,7 @@ export default function AddEventForm() {
           Gruppenmitglied hinzufügen
         </Button>
 
-        <ListUser />
+        <ListUsers />
```

Comment:
Imho `MembersList` would be a better name for a component, the verb oriented name `ListUsers` would be better suited for a function that does something.